services/explorer/src/theme-provider/ExplorerThemeProvider.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { FC } from 'react';
-import { ThemeProvider } from '@mui/material/styles';
-import { theme } from './theme';
 import { StylesProvider, createGenerateClassName } from '@mui/styles';

```

Comment:
Maybe we can also get rid of `mui/styles`

Reply:
@rattrap on #513 I was tinkering most of the morning to shadow it from @taraxa-ui, but it seems that typings are not portable because the `makeStyles` is dynamically defining the return type and we'd need to infer to it anyways via

```TypeScript
: keyof Props extends never
  ? // `makeStyles` where the passed `styles` do not depend on props
    (props?: any) => ClassNameMap<ClassKey>
  : // `makeStyles` where the passed `styles` do depend on props
    (props: Props) => ClassNameMap<ClassKey>;
```

I mean it works, but we lose the type definitions...